libs/standard-tests/langchain_tests/integration_tests/chat_models.py

```diff
@@ -74,7 +74,7 @@ def _validate_tool_call_message_no_args(message: BaseMessage) -> None:
 
 class ChatModelIntegrationTests(ChatModelTests):
     @property
-    def standard_chat_model_params(self) -> dict:
+    def _standard_chat_model_params(self) -> dict:
```

Comment:
is there a way to configure this via docstring, or does it have to be by name? just to avoid having to update all the implementations

Reply:
From what I can tell the only integration that implements it is AWS. I added `:meta private:` to docstrings but it still seems to show up in the attributes table. Going with that for now.

I'm actually unclear on what is intended for this property. It is always combined with `chat_model_params`, so my guess was that we were testing a set of standard params (temperature, max_tokens, etc) that we want all models to support, and users should only overwrite `chat_model_params`. If that's correct, then we should update AWS to not override `standard_chat_model_params`.

Otherwise, we should document when/why this should be overwritten (as opposed to specifying things in `chat_model_params`).